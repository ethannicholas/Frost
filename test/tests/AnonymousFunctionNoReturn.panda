class AnonymousFunctionNoReturn {
    @class
    method main() {
        def f := function(x:Int) { }
    }
}