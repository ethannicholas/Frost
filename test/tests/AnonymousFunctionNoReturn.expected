AnonymousFunctionNoReturn.panda:4:18: error: functions must have a return type
1 error
